tests/std/tests/P0355R7_calendars_and_time_zones_io/test.cpp

```diff
@@ -147,15 +147,15 @@ void test_parse(const CharT* str, const CharT* fmt, Parsable& p, type_identity_t
     basic_stringstream<CharT> sstr{str};
     if (abbrev) {
         if (offset) {
-            sstr >> parse(basic_string<CharT>{fmt}, p, *abbrev, *offset);
+            sstr >> parse(fmt, p, *abbrev, *offset);

```

Comment:
These changes leave only three fairly narrow tests that call `parse` with a `basic_string`. There's already a test, `parse_wchar` that's intended to provide basic testing of different character types. It may be worthwhile to try to generalize it to work with all combinations of narrow/wide and raw/`basic_string` format strings. The `WIDEN` macro at the beginning of this file might be a good starting point, but then you'd have to figure out how to either keep that as a raw string or convert it to a `basic_string`.